besu/src/main/java/org/hyperledger/besu/controller/TransitionBesuControllerBuilder.java

```diff
@@ -205,6 +206,8 @@ protected ConsensusContext createConsensusContext(
       final Blockchain blockchain,
       final WorldStateArchive worldStateArchive,
       final ProtocolSchedule protocolSchedule) {
+    preMergeBesuControllerBuilder.postMergeContext(postMergeContext);
+    mergeBesuControllerBuilder.postMergeContext(postMergeContext);
```

Comment:
Please note this change. I couldn't find a better way to get around this and I wonder if there's a better way